packages/components/src/components/ErrorBoundary/ErrorBoundaryScreen.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { FallbackProps } from "react-error-boundary";
+import { View, SafeAreaView } from "react-native";
+import { Button } from "../Button";
+import { EDSStyleSheet } from "../../styling";
+import { useStyles } from "../../hooks/useStyles";
+import { Typography } from "../Typography";
+
+export const ErrorBoundaryScreen = ({ resetErrorBoundary }: FallbackProps) => {
+    const styles = useStyles(theme);
+    return (
+        <SafeAreaView style={styles.safeAreaContainer}>
+            <View style={styles.container}>
+                <Typography color={styles.title_color.color} style={styles.text}>
+                    Error
+                </Typography>
+                <Typography style={styles.text}>
```

Comment:
If you are thinking about line-height, it looks like `body_short` has a line height of 20px. I don't know what the equivalent `Typography` variant would be

Reply:
https://eds.equinor.com/0b0c666ab/p/721544-typography/b/32c137

Reply:
My point is just that we should create variants / groups instead of hard-coding values like this. If this is body_short, then we need a body short variant.

 Also, You can check if this still holds, but react native seems to have a piss-old bug that causes line heights with Text to display incorrectly, so all current Typography variants has removed this until they fix it.

Reply:
The font is changed body_long! (sorry for changing, but i think this makes it more readable)